app/src/main/java/mozilla/lockbox/presenter/SettingPresenter.kt

```diff
@@ -116,23 +109,27 @@ class SettingPresenter(
             )
         )
 
+        var supportIndex = 2
+
         if (fingerprintStore.isFingerprintAuthAvailable) {
             settings = listOf(
                 ToggleSettingConfiguration(
-                    title = context.getString(R.string.unlock),
+                    title = R.string.unlock,
                     toggleDriver = Observables.combineLatest(
                         settingStore.unlockWithFingerprintPendingAuth,
                         settingStore.unlockWithFingerprint
                     )
                         .map { unlock -> unlock.first.takeIf { it } ?: unlock.second },
-                    toggleObserver = autoLockObserver
+                    toggleObserver = enableFingerprintObserver
                 )
             ) + settings
+
+            supportIndex += 1
```

Comment:
This is kinda gross, but I think comes out of a consequence of using the `SectionedItemListAdapater`.